cmd/readPipelineEnv.go

```diff
@@ -37,6 +44,20 @@ func runReadPipelineEnv() error {
 		return err
 	}
 
+	// try to encrypt
+	if secret, ok := os.LookupEnv("PIPER_pipelineEnv_SECRET"); ok && secret != "" {
+		log.Entry().Debug("found PIPER_pipelineEnv_SECRET, trying to encrypt CPE")
+		jsonBytes, _ := json.Marshal(cpe)
+		encrypted, err := encrypt([]byte(secret), jsonBytes)
+		if err != nil {
+			log.Entry().Fatal(err)
+		}
+
+		os.Stdout.Write(encrypted)
```

Comment:
Please keep in mind that groovy code (Jenkins) expects JSON, after that JSON is unmarshalled to map: https://github.com/SAP/jenkins-library/blob/e2bf31872b057c3247e61249d85e75b957fa5f82/vars/readPipelineEnv.groovy#L11-L12

Reply:
I think we should either change groovy code or exclude Jenkins from the new scenario